views/newsletters.html

```diff
@@ -22,7 +22,6 @@
       var mailpoet_current_time = <%= json_encode(current_time) %>;
       var mailpoet_schedule_time_of_day = <%= json_encode(schedule_time_of_day) %>;
       var mailpoet_date_display_format = "<%= wp_date_format() %>";
-      var mailpoet_start_of_week = "<%= wp_start_of_week() %>";
```

Comment:
Removed this variable as already included in layout.html with mailpoet_wp_week_starts_on